### ISSUE TITLE
Automation_ACC-255.1 (Level A)_Kyma-dashboard_ Icon_ Provide alternative text for images.

### ISSUE THREAD

**Issue by huda866** [kyma-project/busola#3164]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium 
Affected Check Point:	ACC-264.1, ACC-255.1
Affected UI:	Icon
Short text:	Automation_ACC-255.1 (Level A)_Kyma-dashboard_ Icon_ Provide alternative text for images.

1.2 Application Details
Application Name	Kyma-dashboard
Application URL:	https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview
User:	SSO
Technology:	SAP UI5 
UI5 Version:	1.124.1

1.3 Test Environment Details:
GitHub	https://github.com/kyma-project/busola/issues
Web browser:	Google Chrome - Version 127.0.6533.73 (Official Build) (64-bit)
Operating System:	Windows 11 Enterprise Version 22H2
Screen Reader:	JAWS 2024.2405.83 ILM 64 bit for Windows
2. Screen Flow
•	Login into the application using the URL provided.
•	URL: https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview.

3. Issue 
Synopsis: Provide alternative text for images.

Observed Behavior: 
The hidden svg elements in left navigation region does not have a mechanism that allows an accessible name value to be calculated.

Code Snippet: 
<svg xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><defs><symbol id="sapIllus-Dialog-NoData" viewbox="0 0 160 160" height="160" width="160"><path d="M117.5911,134.2588c-14.412,12.9272-35.6124,17.8836-59.239,6.9176C-28.6934,100.7753,44.68-21.4779,114.4214,41.3617,146.6109,70.3658,141.2415,113.0449,117.5911,134.2588Z" style="fill:var(--sapIllus_BackgroundColor)" class="sapIllus_BackgroundColor"></path><path d="M48.5321,28.9918h54.3954a1.2646,1.2646,0,0,1,.9133.39l9.1579,9.5636,9.4579,9.9a1.2587,1.2587,0,0,1,.3487.87L122.8,119.852a5.088,5.088,0,0,1-2.8417,4.0461l-71.1222.0394a1.855,1.855,0,0,0-.27.02,3.4839,3.4839,0,0,1-2.9974-.9879,5.9319,5.9319,0,0,1-.7836-3.1066L44.766,32.7588Z" style="fill:var(--sapIllus_PatternShadow)" class="sapIllus_PatternShadow"></path><path d="M43.0707,23.6238H97.4392a1.2638,1.2638,0,0,1,.914.3909L107.5111,33.6l9.4576,9.9222a1.2641,1.2641,0,0,1,.3491.8723l-.0058,70.2932a3.7924,3.7924,0,0,1-3.7924,3.792H43.09a3.7855,3.7855,0,0,1-3.7924-3.7813c-.0119-15.2406-.0535-72.0688-.0192-87.2988A3.7853,3.7853,0,0,1,43.0707,23.6238Z" style="fill:var(--sapIllus_ObjectFillColor)" class="sapIllus_ObjectFillColor"></path><path d="M115.7633,45.1577h-18.3a1.2547,1.2547,0,0,1-1.2313-1.2769V24.9032A1.2264,1.2264,0,0,1,98.3335,24L116.634,42.9779A1.2833,1.2833,0,0,1,115.7633,45.1577Z" style="fill:var(--sapIllus_BrandColorSecondary)" class="sapIllus_BrandColorSecondary"></path><path d="M117.284,43.06,98.7413,23.5775a1.8929,1.8929,0,0,0-1.3648-.5857H43.2249a4.4159,4.4159,0,0,0-4.4067,4.4c-.0369,16.7822.0172,84.3246.02,87.1918a4.4143,4.4143,0,0,0,4.4067,4.4078h70.148a4.4181,4.4181,0,0,0,4.4066-4.4189l.0062-70.2056A1.89,1.89,0,0,0,117.284,43.06ZM97.2654,24.3215a1.4576,1.4576,0,0,1,.3237-.0391,1.3136,1.3136,0,0,1,.6445.27L116.28,43.3668l.0245.0255a1.1035,1.1035,0,0,1,.1462,1.0946,1.2053,1.2053,0,0,1-.8957.5049L97.4814,44.99c-.2369.0022-.3744-.1633-.555-.4135a1.8052,1.8052,0,0,1-.1368-.8674c-.011-6.2684-.0214-12.5165-.0044-18.7959A.6562.6562,0,0,1,97.2654,24.3215Zm16.3652,93.67H43.0071a3.1711,3.1711,0,0,1-3.169-3.1641c-.0025-2.8827-.057-70.8027-.02-87.678a3.1722,3.1722,0,0,1,3.169-3.1579h52.818c-.0739.2052,0,20,0,20,.0057,1.38.2975,1.9775,1,2l20,3.3745L116.8,114.8178A3.1749,3.1749,0,0,1,113.6306,117.9918Z" style="fill:var(--sapIllus_StrokeDetailColor)" class="sapIllus_StrokeDetailColor"></path><path d="M126.2809,16.0558l-6,7a1,1,0,1,0,1.5176,1.3017l6-7a1,1,0,1,0-1.5176-1.3017Z" style="fill:var(--sapIllus_Layering1)" class="sapIllus_Layering1"></path><path d="M136.9235,31.5309a.9988.9988,0,0,0-1.2451-.6709l-5.915,1.7724a1,1,0,1,0,.5742,1.9161l5.915-1.7725A1,1,0,0,0,136.9235,31.5309Z".

Expected Behavior: 
Add an appropriate alt attribute for images including poster images of videos that do not contain alt attributes. An alt attribute should be a concise and meaningful replacement for the image. For example, when describing a picture of a red balloon, entering alt="red balloon" will be more helpful than alt="This is a picture of a red balloon". The alt text of the image should be in the language of the page or if the page contains multiple languages the language of the surrounding or any referring content.

________________________________________________________________
Kindly refer to attached error and issue processing document for more details.

Regards,
Huda

**Comment by huda866** [comment]
[Automation_ACC-264.1, ACC-255.1 (Level A)_Kyma-dashboard_ Icon_ Provide alternative text for images.docx](https://github.com/user-attachments/files/16667877/Automation_ACC-264.1.ACC-255.1.Level.A._Kyma-dashboard_.Icon_.Provide.alternative.text.for.images.docx)
[Issue processing-09.2021 (1).docx](https://github.com/user-attachments/files/16667879/Issue.processing-09.2021.1.docx)
